dvc/command/run.py

```diff
@@ -250,6 +251,15 @@ def add_parser(subparsers, parent_parser):
             "This doesn't affect any DVC operations."
         ),
     )
+    run_parser.add_argument(
+        "--no-shell",
+        action="store_true",
+        default=False,
+        help=(
+            "Executes command without shell, directly "
+            "feeding it to subprocess"
```

Comment:
This sounds odd. `subprocess` seems a semi-implementation detail. Maybe there is a better way to describe it?